app/views/layouts/application.html.erb

```diff
@@ -13,5 +13,9 @@
 
   <body>
     <%= yield %>
+     <p class="notice"><%= notice %></p>
```

Comment:
Hey, we have more flash messages here. So they should actually only live here. And actually you should use the partial defined in the devise lecture called flashes.html.erb, have a look for it